Description: Bad packaging form upstream
 How come the maintainer of that code even did wrong imports and went away
 with it?
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2015-03-23

--- python-xstatic-angular-lrdragndrop-1.0.2.1.orig/setup.py
+++ python-xstatic-angular-lrdragndrop-1.0.2.1/setup.py
@@ -1,5 +1,5 @@
 
-from xstatic.pkg import angular_smart_table as xs
+from xstatic.pkg import angular_lrdragndrop as xs
 
 # The README.txt file should be written in reST so that PyPI can use
 # it to generate your project's PyPI page.
